### ISSUE TITLE
Newest version doesn't build anymore

### ISSUE THREAD

**Issue by lukas-wresch** [jbeder/yaml-cpp#1223]
Today I am getting the following error when running make.
```
[  2%] Building CXX object CMakeFiles/yaml-cpp.dir/src/convert.cpp.o
In file included from /home/git/actions-runner/_work/dep/yaml-cpp/src/convert.cpp:3:
/home/git/actions-runner/_work/dep/yaml-cpp/include/yaml-cpp/node/convert.h:20:10: fatal error: version: No such file or directory
 #include <version>
          ^~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/yaml-cpp.dir/build.make:102: CMakeFiles/yaml-cpp.dir/src/convert.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:912: CMakeFiles/yaml-cpp.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
```
I believe this is due to commit 6262201182a0ee357d8a456e6d1a8e5984e4c6dd

**Comment by lukas-wresch** [comment]
`__cpp_lib_string_view` requires `<version>` which in turn seems to require C++ 20.
This seems to defeat the purpose of PR #1222

**Comment by jbeder** [comment]
Interesting. What version compiler are you running?

**Comment by lukas-wresch** [comment]
I am still forced to use gcc 8.3.0

**Comment by tchaikov** [comment]
@lukas-wresch hi Lukas, sorry for the regression. should be fixed by #1225.

**Comment by lukas-wresch** [comment]
[tchaikov](https://github.com/tchaikov) Thank you!
I can confirm that this issue is fixed by PR #1225
